docs/proposals/PROPOSAL_H.md

```diff
@@ -0,0 +1,271 @@
+# Proposal H - compatibility artifact in flat format
+
+The purpose of the compatibility artifact is to describe compatibility requirements for container images to run on a host.
+
+This proposal is based on proposal B, F and G with new ideas, some description is kept unchanged and borrowed from these original proposals.
+
+The design goal is to avoid over-flexible speficication, and avoid abusing the specification.
+
+It is NOT TO store variation of user configuration of applications in the compatibility artifact.
+
+The first part describes the compatibility artifact manifest.
+
+The second part focus on how the compatibility requirements to be described in flat format, including image selection and validation.
+
+The third part provides some examples for easy to follow the ideas.
+
+The fourth part discusses how the comaptibility spec colloborates with NFD community.
+
+The fifth part describes how the compatibility artifact could be linked to the image.
+
+Finally, the last part shows the addressed usage scenarios.
+
+## Artifact Manifest
+
+The artifact manifest contains ONLY one blob.
+
+```json
+{
+  "schemaVersion": "0.1.0",
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.oci.image-compatibility.v1",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/nfd.image-compatibility.spec.v1alpha1+json",
+      "digest": "sha256:4a47f8ae4c713906618413cb9795824d09eeadf948729e213a1ba11a1e31d052",
+      "size": 1710,
+      "annotations": {
+        "name": "profileH"
+      }
+    }
+  ],
+  "annotations": {
+    "oci.opencontainers.image.created": "2024-03-27T08:08:08Z"
+  }
+}
+```
+
+The new artifact type for image compatibility MUST be referenced in OCI documentation.
+
+The schema version is to use string semver instead of int version, to help tool in content deserialization.
+
+The media type is defined as nfd.image-compatibility.spec, in order to be defined agnostic to cluster management software.
+
+## image compatibility spec
+
+The spec structure is quite simple, a flat json file. NFD means the Kubernetes node feature discovering SIG.
+
+Schema:
+
+- **`compatibilities`** *list of compatibility set*
```

Comment:
Note that this design / key was originally from proposal C and then the one that subsumed it, Proposal D. The others that now have it followed suit. It might make sense to mention D with the others in your introduction.

Reply:
yes, all previous precious work should be credited, also the annotations/platform feature idea.